tmf2/org.lttng.scope.tmf2.views.ui/src/org/lttng/scope/tmf2/views/ui/timeline/widgets/timegraph/toolbar/ZoomInButton.java

```diff
@@ -33,7 +33,7 @@ public ZoomInButton(TimeGraphWidget viewer) {
         setOnAction(e -> {
             // TODO Pivot could be the current time selection if it's in the
             // visible time range.
```

Comment:
This comment can be removed now. It's the zoom() method that will determine which pivot to use.